[release-1.18] Release Notes (#13269)
istio/istio.io: content/en/news/releases/1.18.x/announcing-1.18/_index.md

---
title: Announcing Istio 1.18.0
linktitle: 1.18.0
subtitle: Major Release
description: Istio 1.18 Release Announcement.

> publishdate: 2023-06-02

Reviewer: Noting also you have 06-03 on another page.

release: 1.18.00
aliases:
    - /news/announcing-1.18.0
---

> We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiopeng Han` from . The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.

Reviewer: Do we want to call out stability level of Ambient?

Reviewer 2: I know we have plenty of users asking about eBPF and Ambient. Should we say anything about the licensing issue we are dealing with?

Author: @howardjohn Can you chime in here on Eric and Jacob's questions?

> We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiopeng Han` from . The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.

Reviewer 3: ```suggestion
We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiaopeng Han` from DaoCloud. The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.
```

{{< relnote >}}

{{< tip >}}

> Istio 1.17.0 is officially supported on Kubernetes versions `1.24` to `1.27`.

Reviewer 2: ```suggestion
Istio 1.18.0 is officially supported on Kubernetes versions `1.24` to `1.27`.
```

---
title: Announcing Istio 1.18.0
linktitle: 1.18.0
subtitle: Major Release
description: Istio 1.18 Release Announcement.

> publishdate: 2023-06-02

Reviewer: Still old date, noticed in list of news items.

Author: I'll update the dates across all the files when we have locked in a date later today

release: 1.18.00
aliases:

>     - /news/announcing-1.18.0

Reviewer: ```suggestion
    - /news/announcing-1.18
    - /news/announcing-1.18.0
```

{{< relnote >}}

{{< tip >}}
Istio 1.18.0 is officially supported on Kubernetes versions `1.24` to `1.27`.
{{< /tip >}}

## What's new

### Ambient Mesh
Istio 1.18 marks the first release of ambient mesh, a new Istio data plane mode that’s designed for simplified operations, broader application compatibility, and reduced infrastructure cost. For more details see the [announcement blog](https://istio.io/latest/blog/2022/introducing-ambient-mesh/).

### Gateway API Support Improvements

> Istio 1.18 improves Istio's gateway-API support, including support for extra v1beta1 resources and enhancements to automated deployment logic to no longer rely on pod injection.  Users of Gateway API on Istio should review this release's upgrade notes for important guidance on upgrading.

Reviewer: ```suggestion
Istio 1.18 improves Istio's Gateway-API support, including support for extra v1beta1 resources and enhancements to automated deployment logic to no longer rely on pod injection.  Users of Gateway API on Istio should review this release's upgrade notes for important guidance on upgrading.
```
remove the `-` as well?

Reviewer 3: I think this should be something like `Istio 1.18 improves support for the Kubernetes Gateway API.`? It might be misinterpreted as part of the Istio's API.

---
title: Announcing Istio 1.18.0
linktitle: 1.18.0
subtitle: Major Release
description: Istio 1.18 Release Announcement.
publishdate: 2023-06-05
release: 1.18.00
aliases:
    - /news/announcing-1.18
    - /news/announcing-1.18.0
---

We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiaopeng Han` from DaoCloud. The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.

{{< relnote >}}

{{< tip >}}
Istio 1.18.0 is officially supported on Kubernetes versions `1.24` to `1.27`.
{{< /tip >}}

## What's new

### Ambient Mesh

> Istio 1.18 marks the first release of ambient mesh, a new Istio data plane mode that’s designed for simplified operations, broader application compatibility, and reduced infrastructure cost. For more details see the [announcement blog](/latest/blog/2022/introducing-ambient-mesh/).

Reviewer: ```suggestion
Istio 1.18 marks the first release of ambient mesh, a new Istio data plane mode that’s designed for simplified operations, broader application compatibility, and reduced infrastructure cost. For more details see the [announcement blog](/blog/2022/introducing-ambient-mesh/).
```
